tests/unit/scripting.tcl

```diff
@@ -812,109 +771,100 @@ start_server {tags {"scripting"}} {
     } {} {external:skip}
 }
 
-foreach cmdrepl {0 1} {
-    start_server {tags {"scripting repl needs:debug external:skip"}} {
-        start_server {} {
```

Comment:
maybe it would be easier to review and better for the blame log if you keep the old indentation..